packages/contracts/contracts/Dependencies/IOwnable.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.17;
+
+interface IOwnable {
```

Comment:
I'm not sure about that, could you describe where we need that particular interface?

Reply:
It`s needed for the _transferOwnership function interaction, so it is being applied here: https://github.com/Threshold-USD/dev/blob/3371b2b661ab1d729d168e59114e7555f7221421/packages/contracts/contracts/Interfaces/ITHUSDToken.sol#L9

Reply:
But how it used outside contracts?

Reply:
If I remove the IOwnable interface from the ITHUSDToken, it returns an error in https://github.com/Threshold-USD/dev/blob/3371b2b661ab1d729d168e59114e7555f7221421/packages/contracts/contracts/THUSDOwner.sol#L39

Stating that `Member "transferOwnership" not found or not visible after argument-dependent lookup in contract ITHUSDToken.`, so for calling the `thusdToken.transferOwnership` we need to specify the IOwnable interface for the ownable transferOwnership function.

Reply:
`Ownable(address(thusdToken)).transferOwnership` this should work
thing is `IOwnable` is not full interface, so to make it right there should be all functions not just `transferOwnership` and I think this is too much, so other way - use `THUSDToken` instead of interface in `THUSDOwner` or use wrapping with `Ownable`

Reply:
Based on your suggestion I used `THUSDToken` instead of interface in `THUSDOwner`.

https://github.com/Threshold-USD/dev/pull/108/commits/dddc9fb8217afdaa0cc28df48956c3de1936b384